css/style.css

```diff
@@ -6,3 +6,55 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+  body {
+background-color: rgb(227, 237, 241);
+margin: 2em;
+padding: 1rem;
+display: flex;
+flex-direction: column;
+ }
+
+ h1{
+color: black;
+text-align: center;
+}
+nav{
+    text-align: center;
+    margin: 4rem;
+}
+main{
+    margin: 3em;
+    padding: 1em;
+    padding-left: 23em;
+    display: flex;
+    flex-direction: column;
+}
+p:first-child {
+   margin: auto;
+}
+aside{
+    margin:1em;
+    padding-top: 1em;
+    padding-left: 5em;
+    display: flex;
+    flex-direction: column;
+}
+.column{
+    float: left;
+    width: 50%;
+}
+.row{
+    content: none;
+    display: table;
+    clear: both;
+}
+img{
+    width: 15em;
+
+}
+footer{
+    margin: 2em;
```

Comment:
What would happen if you set the font size on the footer to 2em? Try it and have a think about when em sizing is a a good idea vs rem sizing (or other CSS units)